### PR TITLE
Fixes #64 colin.spec and packaging fixes

### LINKED ISSUE #64: package colin as an RPM and add it to Fedora

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This issue adds colin.spec file and some packaging fixes.
I tested it in real installation process and usage process. (COPR https://copr.fedorainfracloud.org/coprs/phracek/colin/)

### CODE REVIEW

File: colin.spec [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Summary:        Tool to check generic rules/best-practices for containers/images/dockerfiles.
+
+License:        GPLv3+
+URL:            https://github.com/user-cont/colin
+Source0:        https://files.pythonhosted.org/packages/source/c/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+
+%description
+`colin` as a tool to check generic rules/best-practices
```
Comment (jpopelka) [inline colin.spec]: as -> is

File: colin.spec [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+%build
+%py3_build
+
+%install
+%py3_install
+
+%files -n python3-%{pypi_name}
+%license LICENSE
+%doc README.md
+%{_bindir}/%{pypi_name}
```
Comment (TomasTomecek) [inline colin.spec]: I agree with @bkabrda; it would be great to have the executable in colin rpm

Comment (lachmanfrantisek) [reply colin.spec]: So we should split the code into the following parts?

- `python3-colin` (core) (only basic checks)
- `colin` (only executable)
- additional checks+rulesets e.g. `python3-colin-redhat`

Does it mean also different pip packages and different repositories? Or is it only a 'problem' with `rpm` packages?

Comment (phracek) [reply colin.spec]: python3-colin-redhat would be in private repo on our GitLab.

Comment (lachmanfrantisek) [reply colin.spec]: Yes, of course. I've tried to make the package with additional checks/rulesets and it's really easy:
https://github.com/lachmanfrantisek/colin-openhouse/ (just WIP)

Comment (TomasTomecek) [reply colin.spec]: how about `colin-ruleset-redhat`? I'm not sure whether we actually want to have two: one open (our images are open, nothing to hide, really), one closed (for checks which are tied to internal infra)

File: colin.spec [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Version:        0.0.3
+Release:        1%{?dist}
+Summary:        Tool to check generic rules/best-practices for containers/images/dockerfiles.
+
+License:        GPLv3+
+URL:            https://github.com/user-cont/colin
+Source0:        https://files.pythonhosted.org/packages/source/c/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+Requires:       python3-%{pypi_name}
+Requires:       %{pypi_name}-doc
```
Comment (TomasTomecek) [inline colin.spec]: Please drop this requirement, it doesn't make sense.

**Pull request merged**